src/ghastoolkit/octokit/codescanning.py

```diff
@@ -71,9 +72,16 @@ def __str__(self) -> str:
 class CodeScanning:
     """Code Scanning."""
 
-    def __init__(self, repository: Optional[Repository] = None) -> None:
+    def __init__(
+        self,
+        repository: Optional[Repository] = None,
+        retry_count: int = 240,
```

Comment:
I can feel the pain here - a python wrapper that is ultimately tied into actions.  Difficult to create a callback mechanism. Might be worth heavily documenting this one or required input not default :)

Reply:
Agree. Lets me remove the defaults and it is an opt-in feature